cmd/quickstart.go

```diff
@@ -354,19 +360,22 @@ func retryWithSampleSpec(ctx context.Context, workflowFile *workflow.Workflow, i
 	return true, err
 }
 
-func shouldLaunchStudio(ctx context.Context, wf *run.Workflow) bool {
-	// Only one source at a time is supported in the studio at the moment
-	if len(wf.SourceResults) == 1 {
-		// If the source has a linting result, then it was loaded successfully so we can show something in the studio
-		if maps.Values(wf.SourceResults)[0].LintResult != nil {
-			// This will be true in most cases for admins, unless they have `auth switch`ed to a different workspace
-			if config.GetWorkspaceID() == "self" {
-				return true
-			}
-		}
+func shouldLaunchStudio(ctx context.Context, wf *run.Workflow, fromQuickstart bool) bool {
+	// TODO: Remove this when ready to launch for everyone
+	if !config.IsAdminUnsafe() {
+		return false
 	}
 
```

Comment:
I think let's enforce admin for now and have a single pull request for enabling globally

Reply:
That's what this is, no?

Reply:
Ah sorry missed the early return